readme.md

```diff
@@ -24,9 +24,9 @@ npm install cmu-pronouncing-dictionary
 ## Use
```

Comment:
Could you also add an ESM-only section to Install, and an exports one in API?

Similar to https://github.com/syntax-tree/hast-util-sanitize/commit/10769421b29fd85f0482491f1d262f524846544e#diff-5a831ea67cf5cf8703b0de46901ab25bd191f56b320053be9332d9a3b0d01d15.

Reply:
added in f2da122